index.html

```diff
@@ -64,5 +64,183 @@ <h2 class="container job-title">I am a Software developer </h2>
+                    <li class="html">html</li>
+                    <li class="Boot">Bootstrap</li>
+                    <li class="Ruby">Ruby</li>
+                </ul>
+            <div>
+                <button class="project-button" type="button" id="thirdTitle">See Project</button></div>
+            </div>
+        </div>
+    </div>
+ <!-- Fourth-Profesional-Art-Printing-Data -->
+ <div class="card-section">
+    <div class="work-card">
+        <div class="card-content flex-container">
+            <h4 class="title">Profesional Art Printing Data</h4>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+                <li class="html">html</li>
+                <li class="Boot">Bootstrap</li>
+                <li class="Ruby">Ruby</li>
+            </ul>
+            <button class="project-button" type="button"  id="fourthTitle">See Project</button>
+        </div>
+    </div>
+</div>
+
+<!-- Fifth-Profesional-Art-Printing-Data -->
+<div class="card-section">
+    <div class="work-card">
+        <div class="card-content flex-container">
+            <h4 class="title">Profesional Art Printing Data</h4>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+                <li class="html">html</li>
+                <li class="Boot">Bootstrap</li>
+                <li class="Ruby">Ruby</li>
+            </ul>
+        <div>
+    
+            <button class="project-button" type="button" id="fifthTitle">See Project</button></div>
+    
+        </div>
+    </div>
+</div>
+
+<!-- Sixth-Profesional-Art-Printing-Data -->
+<div class="card-section">
+    <div class="work-card">
+        <div class="card-content flex-container">
+            <h4 class="title">Profesional Art Printing Data</h4>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            <ul>
+                <li class="html">html</li>
+                <li class="Boot">Bootstrap</li>
+                <li class="Ruby">Ruby</li>
+            </ul>
+        <div>
+            
+            <button class="project-button" type="button" id="sixthTitle">See Project</button></div>
+        
+        </div>
+    </div>
+</div>
+
+
+</section><!-- #Recent Works -->
+<!-- ======================
+    About me
+    ====================== -->
+    <section class="about-me" id="about-me">
+        <div class="about-text">
+            <h2>About Me</h2>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <button type="button">Get My Resume</button>
+            <hr class="hr">
+        </div>
```

Comment:
- Please add the background images for this section as it is in the Figma design.
- Please add padding on the button `Get My Resume` so that it looks like that of the Figma design
##### Figma
![Screenshot from 2022-06-04 10-06-03](https://user-images.githubusercontent.com/58771507/171992689-f73b3836-0ea1-427f-aa4e-3e7b7e2b0473.png)

##### Your design 
![Screenshot from 2022-06-04 10-06-36](https://user-images.githubusercontent.com/58771507/171992709-b132f13a-b939-43e0-aca9-20f3db61d4ba.png)